OfficeToPDF/Program.cs

```diff
@@ -1087,6 +1088,10 @@ Office application
   /powerpoint_output <type> - Controls what is generated by output. Possible values are slides, notes,
                               outline, build_slides, handouts and multi-page handouts using handout2,
                               handout3, handout4, handout6 and handout9. The default is slides.   
+  /powerpoint_ref_fonts     - Specifies whether to include a bitmap of the text. If False, the font is referenced, 

```

Comment:
You could use the same description as used for `/word_ref_fonts` as it doing the same thing.